HW_module_3.py

```diff
@@ -0,0 +1,44 @@
+import re
+import string
+
+message = """homEwork:
+	tHis iz your homeWork, copy these Text to variable. 
+	You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+	it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE. 
+	last iz TO calculate nuMber OF Whitespace characteRS in this Text. caREFULL, not only Spaces, but ALL whitespaces. I got 87.
+"""
+
+# Last words of each existing sentence (whitespace+word+dot)
+find_all_last_word = re.findall(r'\s(\w+)?\.', message)
+
+variable = find_all_last_word[0]
+view = find_all_last_word[1]
+paragraph = find_all_last_word[2]
+here = find_all_last_word[3]
+mistake = find_all_last_word[4]
+text = find_all_last_word[5]
+whitespaces = find_all_last_word[6]
+number_87 = find_all_last_word[7]
```

Comment:
Moderate: Why do you need all of them?
The same can be achieved by: 
f'\t{", ".join(find_all_last_word)}.\n'